packages/core/stories/interactable-card/interactable-card.stories.tsx

```diff
@@ -0,0 +1,332 @@
+import { ChangeEvent, useState } from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import {
+  H3,
+  Text,
+  StackLayout,
+  RadioButton,
+  RadioButtonGroup,
+  Label,
+  InteractableCard,
+  InteractableCardGroup,
+  InteractableCardProps,
+  InteractableCardValue,
+  Avatar,
+  CheckboxIcon,
+  RadioButtonIcon,
+} from "@salt-ds/core";
+
+import { BankIcon, CreditCardIcon, DiamondIcon } from "@salt-ds/icons";
+
+import "./interactable-card.stories.css";
+
+export default {
+  title: "Core/Card/Interactable Card",
```

Comment:
title: "Core/Interactable Card/Interactable Card",

Reply:
I wanted to group it with link card under Card

Reply:
makes sense, i was just thinking bc of the new package folder

Reply:
True true, I think because they are grouped on the site docs then this is ok